base_custom_attribute/tests/test_build_view.py

```diff
@@ -173,3 +196,45 @@ def test_render_all_field_type(self):
                 self.assertEqual(previous.tag, "b")
             else:
                 self.assertFalse(attr.get("nolabel", False))
+
+    # TEST on NATIVE ATTRIBUTES
+    def test_include_native_attr(self):
```

Comment:
The test need to be refactored toooooo long.
First split it (never never test 2 cases in one test), one case per test.

For this case test th emethod field_view_get directly. It will avoid all the logic of deepcopy